app/assets/javascripts/modules/coronavirus-landing-page.js

```diff
@@ -42,5 +43,21 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     })
   }
 
+  CoronavirusLandingPage.prototype.addTimelineCountrySelector = function () {
+    var timelineRadios = document.querySelector('.js-change-location')
+    if (timelineRadios) {
+      timelineRadios.addEventListener('submit', function (e) {
+        e.preventDefault()
+      })
+      timelineRadios.addEventListener('change', function (e) {
+        var sections = document.querySelectorAll('.js-covid-timeline')
+        for (var i = 0; i < sections.length; i++) {
+          sections[i].style.display = 'none'
+        }
+        document.getElementById('nation-' + e.target.value).style.display = 'block'
+      })
+    }
```

Comment:
I think this line has changed now although we are using `document.querySelector` elsewhere.

It would be more in the spirit of the module system to scope within the module's $element but in this case that is almost the entire page content, which itself is not in keeping with the module system!

I could initialise this particular bit of the code to a specific area of the page but this feels like a bit of a sticking plaster at this point. I think it might be more sensible to revisit this whole JS file to see if we can improve it separately, I'll make a note.